docs/faq.md

```diff
@@ -112,53 +96,36 @@ In Bitcoin Cold storage we can use multisig setups and different hardware wallet
+### *How safe is the app to use?*
 
 It is watch-only (private keys are protected by HWW) and compatible with multisig in Electrum, so even if something breaks you always have a fallback option while we fix the bug. So go for it :)
 
 We try to use default descriptors and derivation paths exactly for this reason - to be compatible with other wallets. Would be nice to keep it this way, but at a certain point we will need to diverge - for example when we add miniscript support.
 
-### *What does WIP mean?*
-
-WIP means that we don't try to be very backward-compatible at the moment. At some point we may change wallet storage format for example, and you would need to migrate using some script or create wallets from scratch. In this case, we would provide migration scripts.
-
 ### *What's the difference between Specter-desktop and Specter-DIY?* 
 
-Specter-desktop is a watch-only GUI software wallet running on Bitcoin Core using its wallet and full node functionality.
+Specter-desktop is a watch-only GUI software wallet running on Bitcoin Core or Electrum using its wallet and full node functionality.
 Bitcoin Core tracks addresses, UTXO (unspent transaction outputs) and composes PSBT (partially-signed bitcoin transactions).
 
-Whereas, [Specter-DIY](https://github.com/cryptoadvance/specter-diy) is a do-it-yourself hardware wallet from off the shelf components, that signs and broadcasts transactions using QR codes that forgets your private keys when powered off.
+Whereas, [Specter-DIY](https://docs.specter.solutions/diy/) is a do-it-yourself hardware wallet from off the shelf components, that signs and broadcasts transactions using QR codes that forgets your private keys when powered off.
 
 ### *Is a full node necessary for using Specter-desktop?*
 
-Yes, a Bitcoin node is needed to provide all relevant data without relying on 3rd parties, and also for its watch-only wallet capabilities. However, Specter allows you to easily setup a (pruned-) node easily within Specter. If you can, a full-node is recommended but will take a lot longer to synchronize.
-
-### *Can I use pruned mode?*
-
-Yes, but if you have many older addresses you will need to re-download the blockchain in order to see your balance and transaction history, which will take some time.
-Since v0.8.0 there is a workaround as you can download history from an external blockexplorer which has privacy implications. Make sure to read the tooltip-hints when using that feature and also consider this question in the  [troubleshooting-section](#i-created-an-existing-wallets-but-even-after-rescanning-specter-couldnt-find-any-funds)..
+No, a Bitcoin node is no longer needed. It's encouraged that you use your own Node (or your own electrum-server) but you can also use an external electrum-server in the public internet. Since Version 2 we have two proposals to use easily. As this was too cumbersome to use. we no longer provide an internal node since version 2. Use an external electrum instead.
```

Comment:
```suggestion
No, with Specter 2.0, a Bitcoin node is no longer needed. It's encouraged that you use your own Bitcoin Core node or your own Electrum server but you can also use a public Electrum server. Since version 2.0 we no longer support a Bitcoin Core node managed by the Specter app. You could use the external Electrum option instead or manage the Bitcoin Core node yourself.
```